atspi-common/README.md

```diff
@@ -5,3 +5,19 @@ This crate is meant to only contain the absolute lowest common denominator for l
 therefore, it attempts to be able to compile on basically any architecture (including WASM).
 
 Please use the internal documentation to learn how to use these data structures.
+
+## Compiling for WASM
```

Comment:
How error serialization and Wasm support are related? Please make the job of reviewers easier by keeping your PRs focused on one new feature.

Thanks.

Reply:
To send Rust types across WASM FFI boundaries, tou need to e bl to serialize to some stable gormat, then deserialize on the other side.